design/resources.go

```diff
@@ -196,6 +196,24 @@ var _ = a.Resource("identity", func() {
 		a.Response(d.BadRequest, JSONAPIErrors)
 		a.Response(d.InternalServerError, JSONAPIErrors)
 	})
+
+	a.Action("show", func() {
+		a.Security("jwt")
```

Comment:
This can't be secured. We allow non logged in users to see WorkItems, which means they can't see who things are assigned to etc.

When we get more info about the Users we might choose to show more based on being logged in / permissions etc.